Cargo.toml

```diff
@@ -24,5 +24,6 @@ members = [
     "examples/private_xo",
     "examples/gameroom/database",
     "examples/gameroom/daemon",
+    "examples/cm-repl/daemon",
```

Comment:
Could you give some more context in this commit message?  Is this a daemon process that provides the functionality to a REPL or the REPL itself?  Normally I would think of a CLI tool, so it's not quite clear.

Reply:
The intent of this request was to write up a description of the server.  The line you put in the description would have worked just as fine for a summary line.  The message should include more information about the repl server, what it does, as of this commit etc.

Reply:
Likewise, the cli commit could use a similar description.

Reply:
Updated commit messages, let me know if they're to your liking, I don't want phone anything in :)

Reply:
These are great!